testdata/range_del

```diff
@@ -1282,7 +1282,7 @@ compact f-f L0
 
 compact a-f L1
 ----
-2: a-c c-f
+2: a-c f-f
```

Comment:
This change doesn't seem right. Still thinking about it.

Reply:
Never mind, I think it is right. The range tombstone is only in the `a-c` L1 file from the previous step, and we're verifying that file doesn't during L1->L2. The `f-f` file isn't supposed to contain the range tombstone, so actually the outcome before, where it got expanded to `c-f`, didn't make sense.